Sources/PostgresNIO/New/Messages/Password.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 06.01.21.
+//
+
+extension PSQLFrontendMessage {
```

Comment:
Ditch Xcode template headers.
```suggestion
extension PSQLFrontendMessage {
```